min-required.txt

```diff
@@ -1,6 +1,6 @@
-lxml==2.1.0  # >=1.3.4 should work
+lxml==3.5.0
```

Comment:
@dwaynebailey see the `requirements` section here http://lxml.de/installation.html

we need to check those against debian wheezy,wheezy-backports,jessie and some flavours of ubuntu i reckon